include/ttmlir/Target/TTNN/program.fbs

```diff
@@ -106,10 +101,21 @@ enum EltwiseOpType: uint32 {
   Gelu = 36,
   LogicalXor = 37,
   Clamp = 38,
+  LeakyRelu = 39,
+}
+
+table ClampOpParams {
+  min: float;
+  max: float;
+}
+
+table ElementwiseOpWithFloatParams {
```

Comment:
Nit: `EltwiseOpWithFloatParams` just to align with us using eltwise in other places.